frontend/src/lib/components/common/form/Field.svelte

```diff
@@ -1,6 +1,8 @@
 <script lang="ts">
   export let customStyle: string | undefined = undefined;
   export let bgColor: string = 'bg-base-100';
+  export let id: string | undefined = undefined;
```

Comment:
We'll likely need to overhaul these later but for the time being, maybe add more robust typing such that `id` and `label` must both be provided or neither.